### PR TITLE
Refactor Data Integration for Bridge-V2

### PR DESCRIPTION
**Pull request by Meggielqk**

### CODE REVIEW

**Comment by Meggielqk** [comment]
Still in draft state and more data integration pages will be updated later...

File: en_US/data-integration/data-bridge-cassa.md [hunk 1]
```diff
@@ -102,25 +118,25 @@ This section demonstrates how to create a rule to specify the data to be stored
...
    ```
 
-8. Advanced settings (optional):  Choose whether to use **sync** or **async** query mode as needed. For details, see [Data Integration](./data-bridges.md).
+9. Advanced settings (optional):  Choose whether to use **sync** or **async** query mode as needed. For details, see [Data Integration](./data-bridges.md).
 
-9. Click the **Add** button to complete the Sink configuration. Back on the **Create Rule** page, you will see the new Sink appear under the **Action Outputs** tab.
+10. Click the **Add** button to complete the Sink configuration. Back on the **Create Rule** page, you will see the new Sink appear under the **Action Outputs** tab.
 
-10. On the **Create Rule** page, verify the configured information and click the **Create** button to generate the rule. The rule you created is shown in the rule list and the **status** should be connected.
+11. On the **Create Rule** page, verify the configured information and click the **Create** button to generate the rule. The rule you created is shown in the rule list and the **status** should be connected.
```
Comment (zmstone) [inline en_US/data-integration/data-bridge-cassa.md:139]: markdown allows you to use `1. ` for all bullets, and it will be rendered as 1 ~ N.

Comment (zmstone) [reply en_US/data-integration/data-bridge-cassa.md:139]: so there is no need to manually keeping the order

File: en_US/data-integration/data-bridge-mysql.md [hunk 2]
```diff
@@ -94,11 +94,28 @@ use emqx_data;
...
    ```
 
-## Create Rule and MySQL Sinks
+## Create a Connector
 
-MySQL Sinks for message storage and event recording require different SQL templates. Therefore, you need to create 2 different Sinks for message storage and event recording. 
+This section demonstrates how to create a Connector to connect the Sink to the MySQL server.
 
-### Message Storage
+The following steps assume that you run both EMQX and TDengine on the local machine. If you have TDengine and EMQX running remotely, adjust the settings accordingly.
```
Comment (wivwiv) [inline en_US/data-integration/data-bridge-mysql.md]:
```suggestion
The following steps assume that you run both EMQX and MySQL on the local machine. If you have MySQL and EMQX running remotely, adjust the settings accordingly.
```

**Review by wivwiv — CHANGES_REQUESTED** [review]

**Review by wivwiv — APPROVED** [review]

**Pull request merged**